mbof/views.py

```diff
@@ -42,16 +44,28 @@ class CurrentUserViewSet(UserViewSet):
     queryset = User.objects.filter(loginName=os.getenv('REMOTE_USER')).order_by('surname')
 
 
-class MessageViewSet(viewsets.ModelViewSet):
+class EventViewSet(viewsets.ModelViewSet):
     """
     API endpoint that allows users to be viewed or edited.
     """
-    queryset = Message.objects.all().order_by('postingTime')
-    serializer_class = MessageSerializer
+    queryset = Event.objects.all().order_by('postingTime')
+    serializer_class = EventSerializer
+
+
+class CurrentEventViewSet(viewsets.ModelViewSet):
```

Comment:
I made an experiment in [/tl-its-umich-edu/hacks_mbof/blob/master/mbof/views.py#L65](/tl-its-umich-edu/hacks_mbof/blob/master/mbof/views.py#L65) you should see.  You could make `CurrentEventViewSet` subclass `EventViewSet` instead of `viewsets.ModelViewSet`.  It will then inherit `serializer_class` and you'll only need to define `get_queryset()`.